### ISSUE TITLE
Observe Service Catalog stability 

### ISSUE THREAD

**Issue by PK85** [kyma-project/kyma#518]
AC:
- I want to see that Service Catalog components have been restarted. I will be able to use that information in comparison with SC related test failures
- Solution needs to be generic, for example could handle kubeless component check

**Comment by PK85** [comment]
Check if pod-watcher can be used here.

**Comment by mszostok** [comment]
Status:
I've checked the community, and there is no tool which we can use for such case. Possible tools to use in the future:
- [kubewatch](https://github.com/bitnami-labs/kubewatch/tree/v0.0.4) -but this tool is in the early stage. There will be an option to watch pods/deployments with given reason, but for now, this feature is [commented](https://github.com/bitnami-labs/kubewatch/blob/v0.0.4/config/config.go#L55). This tool supports a lot of notifiers out-of-the-box.
- [searchlight](https://github.com/appscode/searchlight) tool which alerts you when things go wrong. There is an option to create an alert for [Pod status](https://appscode.com/products/searchlight/7.0.0/guides/pod-alerts/pod-status/). This tool supports a lot of notifiers out-of-the-box. This tool can be also used for checking CA Cert status, Events, Node status, Pod, Volumes etc. but this requires the installation of the operator for all that functionality.

We decided to change the scope of this issue, and create a new repo under the kyma-incubator where we will put the reproducible test-cases for service-catalog.
